README.md

```diff
@@ -35,3 +35,19 @@ var expected = JToken.Parse(@"{ ""key2"" : ""value"" }");
 actual.Should().BeEquivalentTo(expected);
 ```
 
+You can also use `EquivalencyAssertionOptions<>` with `shouldBeEquivalentTo` assertions, which contains helper methods that you can use to specify the way you want to compare specific data types.
+
+Example:
+
+```c#
+using FluentAssertions;
+using FluentAssertions.Json;
+using Newtonsoft.Json.Linq;
+
+... 
+var actual = JToken.Parse(@"{ ""value"" : 1.5 }");
+var expected = JToken.Parse(@"{ ""value"" : 1.4 }");
+actual.Should().BeEquivalentTo(expected, options => new EquivalencyAssertionOptions<object>()
```

Comment:
Why do you need to create an instance of `EquivalencyAssertionOptions<object>()`? Isn't `options => options.Using(....` enough?